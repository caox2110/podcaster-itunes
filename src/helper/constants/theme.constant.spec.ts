import { themeConstant } from './theme.constant';

const constants = {
  MODE: {
    LIGHT: 'light',
    DARK: 'dark',
  },
  FONT_PRIMARY: "'DM Sans', sans-serif",
};
describe('Theme constant vars', () => {
  it('Should be a defined constant vars for the theme', () => {
    expect(themeConstant).toStrictEqual(constants);
    expect(themeConstant).toMatchSnapshot({
      MODE: {
        LIGHT: expect.stringMatching(/light/),
        DARK: expect.stringMatching(/dark/),
      },
      FONT_PRIMARY: expect.any(String),
    });
  });
});
