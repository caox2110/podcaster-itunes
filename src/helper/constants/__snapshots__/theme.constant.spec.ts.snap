// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Theme constant vars Should be a defined constant vars for the theme 1`] = `
Object {
  "FONT_PRIMARY": Any<String>,
  "MODE": Object {
    "DARK": StringMatching /dark/,
    "LIGHT": StringMatching /light/,
  },
}
`;
